doc/source/whatsnew/v0.18.1.txt

```diff
@@ -228,6 +228,7 @@ Other Enhancements
 - ``pd.read_csv()`` now supports ``delim_whitespace=True`` for the Python engine (:issue:`12958`)
 - ``pd.read_csv()`` now supports opening ZIP files that contains a single CSV, via extension inference or explict ``compression='zip'`` (:issue:`12175`)
 - ``pd.read_csv()`` now supports opening files using xz compression, via extension inference or explicit ``compression='xz'`` is specified; ``xz`` compressions is also supported by ``DataFrame.to_csv`` in the same way (:issue:`11852`)
+- ``pd.read_csv()`` now supports duplicate column names in a file (:issue:`7160`, :issue:`9424`)
```

Comment:
Sure thing.  I was leaving it in `0.18.1` FTTB because it was avoiding merge conflicts.